scummvm.appdata.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>scummvm.desktop</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0+</project_license>
+  <name>ScummVM</name>
+  <summary>Interpreter for several adventure games</summary>
+  <description>
+    <p>
+      ScummVM is an interpreter that will play graphic adventure games written
+      for LucasArts' SCUMM virtual machine (such as Day of the Tentacle and
+      Monkey Island), Sierra's AGI adventures (such as early King's Quest and
+      Space Quest games), Adventure Soft's Simon the Sorcerer 1, 2 and Feeble Files,
+      Revolution Software's Beneath a Steel Sky and Broken Sword 1, 2 and 2.5,
+      Interactive Binary Illusions' Flight of the Amazon Queen,
+      Coktel Vision's Gobliiins, Wyrmkeep's Inherit the Earth, Westwood's
+      Legend of Kyrandia, and various others.
+    </p>
+    <p>
+      This version of ScummVM is sandboxed. You have to copy your games to your
+      Documents directory.
+    </p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image>http://wiki.scummvm.org/images/f/f8/Launcherwithgames.png</image>
+      <caption>ScummVM Launcher</caption>
+    </screenshot>
+  </screenshots>
+  <url type="homepage">https://www.scummvm.org</url>
+  <update_contact>scummvm-devel@lists.scummvm.org</update_contact>
+  <content_rating type="oars-1.1">
+    <content_attribute id="violence-cartoon">none</content_attribute>
```

Comment:
While the application does not contain anything related to violence or sexuality, the games can. Not sure how we should handle it.

Reply:
Yup, I was thinking about this too. Probably a good question for @hughsie?

Reply:
It's a tricky one. On one hand you could think of an emulator like a web browser, as in you could do bad things, or you could do good. In other ways it's just a way of launching games (like steam). Logically the OARS rating would be on the game itself, rather than the emulator. I guess for this case you could chose something plausible (i.e no mature .roms, and nothing that allows player-to-player chat) as a pragmatic way forward.

Reply:
Thanks for your answer! So I think I'll leave the OARS rating as it is now.

Reply:
Maybe add a new content_attribute to OARS for software that is some kind of "platform" for 3rd-party content?